### ISSUE TITLE
Untracked flag usages in daa instruction

### ISSUE THREAD

**Issue by theNestruo** [santiontanon/mdlz80optimizer#17]
(this issue is very similar to https://github.com/santiontanon/mdlz80optimizer/issues/16)

The instruction `daa` requires the flags to be properly set/reset to work (just the half-carry flag?). This set-up is usually performed by a `add/adc/dec/inc/neg/sub/sbc` instruction; so if the number is "just read", a fake operation (such as `add a, 0`) must be inserted. Therefore, the suggestion to _"Remove unnecessary add a,#00 (2 bytes, 8 t-states saved)"_ is not valid:

Example code from _PyramidWarp.enhancedplus.asm_:
```
; Prints the room number
	ld	a,(game.current_room)
	add	a,00h
	daa
```
This example code does not behave as expected without the `add a, 0`.

**Comment by santiontanon** [comment]
Oh, thanks for the report! I just checked, and similar issue as with issue #16 , the instruction set file is missing the flags that daa depends on! I'll fix it later tonight as well. I added all the instructions dependencies by hand going through the z80 manual. So, I'm sure there was some "human error" there, and I missed some. Hopefully those are corrected bit by bit, and eventually the file is 100% correct :)
